### PR TITLE
feat: video listing screen

### PR DESCRIPTION
**Pull request by rahulbakshi4**
Page Added :
- Video Listing page 

Components Added:
- Side bar
- Video card
- Video List Container

To do : Add functionalities

### CODE REVIEW

File: src/frontend/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                        <a className="links">Home</a>
+                    </li>
+                    <li>
+                        <span className='material-icons'>explore</span>
+                        <a className="links">Explore</a>
+                    </li>
+                    <li>
+                        <span className='material-icons'>playlist_add</span>
+                        <a className="links">Playlists</a>
+                    </li>
```
Comment (shrey27) [inline src/frontend/components/Sidebar/Sidebar.js:19]: replace a tags with Link tags

**Review by shrey27 — APPROVED** [review]
overall code looks good
PR is ready to merge

File: src/frontend/components/VideoCard/VideoCard.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                <img class="card-img" src={`https://i.ytimg.com/vi/${id}/maxresdefault.jpg`} alt="card image" />
+            </div>
+            <ul class="stacked-list">
+                <li class="stacked-items">
+                    <img src="https://picsum.photos/id/1012/200" class="avatar avatar-small" alt="user avatar" />
+                    <div class="list-content">
+                        <p class="text-normal fw-semibold">{title}</p>
+                        <p class="text-sm">{creator}</p>
+                    </div>
+                    <button class="icon-btn material-icons stacked-btn">more_vert</button>
```
Comment (OmKakatkar) [inline src/frontend/components/VideoCard/VideoCard.js]: use `className` instead of `class`

File: src/frontend/components/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                    <li>
+                        <span className='material-icons'>history</span>
+                        <a className="links" >History</a>
+                    </li>
+                    <li>
+                        <span className='material-icons'>watch_later</span>
+                        <a className="links">Watch Later</a>
+                    </li>
+
```
Comment (OmKakatkar) [inline src/frontend/components/Sidebar/Sidebar.js:32]: Replace the `a` tag with `Link` or `NavLink` depending on your use case

**Review by OmKakatkar — APPROVED** [review]
Overall looks good

Like:
* Code modularity
* Meaningful variable names

Suggestions:
* Add a preview link for the PR, it is helpful to test the functionality
* Left a comment below, do have a look

File: src/frontend/components/VideoCard/VideoCard.js (hunk 2 above)
Comment (rahulbakshi4) [reply src/frontend/components/VideoCard/VideoCard.js]: Will change them to className

File: src/frontend/components/Sidebar/Sidebar.js (hunk 3 above)
Comment (rahulbakshi4) [reply src/frontend/components/Sidebar/Sidebar.js:32]: I'm yet to add more pages and routes so Will give all of them a `Link` tag

**Pull request merged**